components/landlord/LandlordPage.tsx

```diff
@@ -204,14 +208,15 @@ const LandlordPage = ({ landlord, data }: IProps) => {
 											) : null} */}
 
 											<p className='mt-3 space-y-6 text-sm text-gray-500'>
-												{review.review}
+											{review.review}
 											</p>
 										</div>
 									</div>
 								</div>
 							)
 						})}
 					</div>
+					{otherLandlords.length > 0 && <OtherLandlordInfo data={data}/>}
```

Comment:
Maybe we should cap this at a certain number. Don't want to show 1000 landlords for a big city. Perhaps we can randomly grab 10 from the array if there are more than 10?

Reply:
Agreed - also the SQL query should reflect this too, so only grabbing top 10 rather than dumping out entire DB.

Reply:
Agreed! I was thinking today that I would find a way to limit the query so we aren't wasting overhead on data we won't use.

Reply:
I just committed a change attempting to address this. Hitting a weird bug where the number of "otherlandlords" being rendered is always -2 from the limit i set in the SQL query. Right now, the limit is set to 10 but only 8 will render. If I set it to 12, 10 will render but I would like to understand why that is happening. Will continue working on this weekend but am interested if the bug repeats in a different environment.